src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -878,6 +879,17 @@ public function providePathsForMakePathRelative()
             array('C:/aa/bb/../../cc', 'C:/aa/../dd/..', 'cc/'),
             array('C:/../aa/bb/cc', 'C:/aa/dd/..', 'bb/cc/'),
             array('C:/../../aa/../bb/cc', 'C:/aa/dd/..', '../bb/cc/'),
+            array('aa/bb', 'aa/cc', '../bb/'),
```

Comment:
@chalasr the function always supported relative paths (before 3.2.7). There are even pre-existing test cases for that.